lib/src/builder/generators/insert_generator.dart

```diff
@@ -147,10 +147,12 @@ class InsertGenerator {
       }
     }
 
+    var constructorParameters =
+        requestFields.map((f) => '${f.key.endsWith('?') ? '' : 'required '}this.${f.value}').join(', ');
     return '''
       ${table.annotateWith ?? ''}
       class $requestClassName {
-        $requestClassName({${requestFields.map((f) => '${f.key.endsWith('?') ? '' : 'required '}this.${f.value}').join(', ')},});
+        $requestClassName(${constructorParameters.isNotEmpty ? '{$constructorParameters,}' : ''});
```

Comment:
```suggestion
        $requestClassName($constructorParameters);
```

Reply:
I won't implement this as this changes the constructor from taking named parameters to taking positional ones, which isn't what we want.

Reply:
Oh yes, I missed that.